pkg/machines/components/networks/createNetworkDialog.jsx

```diff
@@ -0,0 +1,560 @@
+                        {_("Any Physical Device")}
+                    </Select.SelectEntry>
+                    <Select.SelectDivider />
+                    <optgroup key="Physical Devices" label="Physical Devices">
+                        { devices.map(dev => {
+                            return (
+                                <Select.SelectEntry data={dev} key={dev}>
+                                    {dev}
+                                </Select.SelectEntry>
+                            );
+                        })}
+                    </optgroup>
+                </Select.Select>
+                { validationState == 'error' &&
+                <HelpBlock>
+                    <p className='text-danger'>{validationFailed.physicalDevice}</p>
+                </HelpBlock> }
+            </FormGroup>
+        </React.Fragment>
+    );
+};
+
+const IpRow = ({ onValueChanged, dialogValues }) => {
+    return (
+        <React.Fragment>
+            <label className='control-label'>
+                {_("IP Configuration")}
+            </label>
+            <Select.Select id='create-network-ip-configuration'
+                           initial={dialogValues.ip}
+                           onChange={value => onValueChanged('ip', value)}>
+                { (dialogValues.forwardMode === "none") &&
+                <Select.SelectEntry data='None' key='None'>
+                    {_("None")}
+                </Select.SelectEntry>}
+                <Select.SelectEntry data='IPv4 only' key='IPv4 only'>
+                    {_("IPv4 only")}
+                </Select.SelectEntry>
+                <Select.SelectEntry data='IPv6 only' key='IPv6 only'>
+                    {_("IPv6 only")}
+                </Select.SelectEntry>
+                <Select.SelectEntry data='IPv4 and IPv6' key='IPv4 and IPv6'>
+                    {_("IPv4 and IPv6")}
+                </Select.SelectEntry>
+            </Select.Select>
+        </React.Fragment>
+    );
+};
+
+const DhcpRow = ({ ipVersion, rangeStart, rangeEnd, expanded, onValueChanged, validationFailed }) => {
+    const validationStart = validationFailed['ipv' + ipVersion + 'DhcpRangeStart'] ? 'error' : undefined;
+    const validationEnd = validationFailed['ipv' + ipVersion + 'DhcpRangeEnd'] ? 'error' : undefined;
+
+    return (
+        <React.Fragment>
+            <label className='checkbox-inline'>
+                <input id={'network-ipv' + ipVersion + '-dhcp'}
+                    type='checkbox'
+                    checked={expanded}
+                    onChange={e => onValueChanged('ipv' + ipVersion + 'DhcpEnabled', !expanded)} />
+                {_("Set DHCP Range")}
+            </label>
+
+            {expanded && <React.Fragment>
+                <div className='create-network-dialog-grid'>
+                    <div className='ct-form'>
+                        <label className='control-label'> {_("Start")} </label>
```

Comment:
htmlFor. I will not mention this comment again. Please adjust similarly or missing htmlFor attrs.